### PR TITLE
Address style issues

### PR DESCRIPTION
**Pull request by TroyWalshProf**

### CODE REVIEW

File: jmaqs-base/src/main/java/com/magenic/jmaqs/base/ConcurrentManagerHashMap.java [hunk 1]
```diff
@@ -11,6 +11,8 @@
  */
 public class ConcurrentManagerHashMap extends ConcurrentHashMap<String, BaseTestObject> {
 
+  private static final long serialVersionUID = 1;
```
Comment (jonreding2010) [inline jmaqs-base/src/main/java/com/magenic/jmaqs/base/ConcurrentManagerHashMap.java:14]: comment for what this variable is used for

File: jmaqs-base/src/main/java/com/magenic/jmaqs/base/ManagerDictionary.java [hunk 2]
```diff
@@ -10,16 +10,18 @@
 /**
  * Driver manager dictionary.
  */
-public class ManagerDictionary extends HashMap<String, DriverManager> implements AutoCloseable {
+public class ManagerDictionary extends HashMap<String, DriverManager<?>> implements AutoCloseable {
+
+  private static final long serialVersionUID = 1;
```
Comment (jonreding2010) [inline jmaqs-base/src/main/java/com/magenic/jmaqs/base/ManagerDictionary.java:15]: add comment for what this variable does

File: jmaqs-utilities/src/main/java/com/magenic/jmaqs/utilities/helper/exceptions/ExecutionFailedException.java [hunk 3]
```diff
@@ -2,11 +2,13 @@
 
 public class ExecutionFailedException extends Exception {
 
+    private static final long serialVersionUID = 1;
```
Comment (jonreding2010) [inline jmaqs-utilities/src/main/java/com/magenic/jmaqs/utilities/helper/exceptions/ExecutionFailedException.java:5]: comment about what this variable does

File: jmaqs-utilities/src/main/java/com/magenic/jmaqs/utilities/helper/exceptions/FunctionException.java [hunk 4]
```diff
@@ -8,6 +8,8 @@
  * Custom FunctionException, as you can't throw a generic exception.
  */
 public class FunctionException extends Exception {
+  private static final long serialVersionUID = 1;
```
Comment (jonreding2010) [inline jmaqs-utilities/src/main/java/com/magenic/jmaqs/utilities/helper/exceptions/FunctionException.java:11]: comment what this variable does

File: jmaqs-utilities/src/main/java/com/magenic/jmaqs/utilities/helper/exceptions/TimeoutException.java [hunk 5]
```diff
@@ -5,9 +5,13 @@
 package com.magenic.jmaqs.utilities.helper.exceptions;
 
 /**
- * Custom TimeoutException, as the built in one does not allow for a passed in inner exception.
+ * Custom TimeoutException, as the built in one does not allow for a passed in
+ * inner exception.
  */
 public class TimeoutException extends Exception {
+
+  private static final long serialVersionUID = 1;
```
Comment (jonreding2010) [inline jmaqs-utilities/src/main/java/com/magenic/jmaqs/utilities/helper/exceptions/TimeoutException.java:13]: comment what this variable does

File: jmaqs-base/src/main/java/com/magenic/jmaqs/base/ConcurrentManagerHashMap.java (hunk 1 above)
Comment (jason-edstrom) [reply jmaqs-base/src/main/java/com/magenic/jmaqs/base/ConcurrentManagerHashMap.java:14]: More importantly, why is it there?  The field is `private static final` and it is not used anywhere in the class.

File: jmaqs-base/src/main/java/com/magenic/jmaqs/base/ManagerDictionary.java (hunk 2 above)
Comment (jason-edstrom) [reply jmaqs-base/src/main/java/com/magenic/jmaqs/base/ManagerDictionary.java:15]: Same question as above.

**Comment by jason-edstrom** [comment]
> Just a couple comments for a variable that seems out of place.
> Also, looks like code coverage took a hit. Might be something to look into

@jonreding2010 The code coverage is only assessing the modified line coverage.  The overall project code coverage will still be in the green by 3-4%. Not worth worrying aobut.

File: jmaqs-base/src/main/java/com/magenic/jmaqs/base/ConcurrentManagerHashMap.java (hunk 1 above)
Comment (jason-edstrom) [reply jmaqs-base/src/main/java/com/magenic/jmaqs/base/ConcurrentManagerHashMap.java:14]: https://stackoverflow.com/questions/30716967/is-java-serialversionuid-of-1l-ok-or-does-it-need-to-be-unique

Comment (jason-edstrom) [reply jmaqs-base/src/main/java/com/magenic/jmaqs/base/ConcurrentManagerHashMap.java:14]: https://stackoverflow.com/questions/30962559/why-java-declared-serialversionuid-in-exception-and-throwable-class

Seems like it has value in exceptions

Comment (jason-edstrom) [reply jmaqs-base/src/main/java/com/magenic/jmaqs/base/ConcurrentManagerHashMap.java:14]: Based on the research, the serialVersionUid is for interoperability throughout the exceptions stack.  Certain aspects of exception will not be able to be thrown if the exception is not serializable.  So serialVersionUid is ok in exceptions and serializble inheritance classes, but are out of place in normal classes.

File: jmaqs-utilities/src/main/java/com/magenic/jmaqs/utilities/helper/Config.java [hunk 6]
```diff
@@ -90,7 +91,7 @@ private Config() {
    * @return A HashMap of the values in the section
    */
   public static Map<String, String> getSection(String section) {
-    HashMap<String, String> sectionValues = new HashMap();
+    HashMap<String, String> sectionValues = new HashMap<String, String>();
```
Comment (jason-edstrom) [inline jmaqs-utilities/src/main/java/com/magenic/jmaqs/utilities/helper/Config.java:94]: `<String,String>` can be replaced with a `<>` operator.

File: jmaqs-utilities/src/main/java/com/magenic/jmaqs/utilities/helper/exceptions/ExecutionFailedException.java (hunk 3 above)
Comment (jason-edstrom) [reply jmaqs-utilities/src/main/java/com/magenic/jmaqs/utilities/helper/exceptions/ExecutionFailedException.java:5]: SerialVeriondUID is acceptable use in an exception

File: jmaqs-utilities/src/main/java/com/magenic/jmaqs/utilities/helper/exceptions/FunctionException.java (hunk 4 above)
Comment (jason-edstrom) [reply jmaqs-utilities/src/main/java/com/magenic/jmaqs/utilities/helper/exceptions/FunctionException.java:11]: SerialVeriondUID is acceptable use in an exception

File: jmaqs-utilities/src/main/java/com/magenic/jmaqs/utilities/helper/exceptions/TimeoutException.java (hunk 5 above)
Comment (jason-edstrom) [reply jmaqs-utilities/src/main/java/com/magenic/jmaqs/utilities/helper/exceptions/TimeoutException.java:13]: SerialVeriondUID is acceptable use in an exception

File: jmaqs-utilities/src/main/java/com/magenic/jmaqs/utilities/helper/Config.java (hunk 6 above)
Comment (TroyWalshProf) [reply jmaqs-utilities/src/main/java/com/magenic/jmaqs/utilities/helper/Config.java]: Done

File: jmaqs-utilities/src/main/java/com/magenic/jmaqs/utilities/helper/exceptions/FunctionException.java (hunk 4 above)
Comment (TroyWalshProf) [reply jmaqs-utilities/src/main/java/com/magenic/jmaqs/utilities/helper/exceptions/FunctionException.java:11]: Keeping per Jason

File: jmaqs-utilities/src/main/java/com/magenic/jmaqs/utilities/helper/exceptions/TimeoutException.java (hunk 5 above)
Comment (TroyWalshProf) [reply jmaqs-utilities/src/main/java/com/magenic/jmaqs/utilities/helper/exceptions/TimeoutException.java:13]: Keeping per Jason

File: jmaqs-base/src/main/java/com/magenic/jmaqs/base/ManagerDictionary.java (hunk 2 above)
Comment (TroyWalshProf) [reply jmaqs-base/src/main/java/com/magenic/jmaqs/base/ManagerDictionary.java:15]: Which variable are we talking about?

Comment (TroyWalshProf) [reply jmaqs-base/src/main/java/com/magenic/jmaqs/base/ManagerDictionary.java:15]: removed

**Comment by jason-edstrom** [comment]
Waiting for final pipeline run to complete before merging.

**Pull request merged**